tools/regionalgam/ab-index.R

```diff
@@ -4,8 +4,10 @@ args = commandArgs(trailingOnly=TRUE)
 source(args[1])
 
 
-tryCatch({input = read.table(args[2], header=TRUE,sep=" ")},finally={input = read.table(args[2], header=TRUE,sep=",")})
-pheno = read.table(args[3], header=TRUE,sep="	")
```

Comment:
what happend to pheno?

Reply:
error from my side ! Thank you @bgruening